.github/renovate.json5

```diff
@@ -0,0 +1,9 @@
+{
+  $schema: "https://docs.renovatebot.com/renovate-schema.json",
+  extends: [
+    "github>kolonialno/renovate-config",
+    "github>kolonialno/renovate-config:autoMerge",
```

Comment:
```suggestion
```

We have next to no test. Before that is fixed, I would suggest we do not auto merge.